lms/djangoapps/program_enrollments/models.py

```diff
@@ -136,6 +137,18 @@ class Meta(object):
     def is_active(self):
         return self.status == ProgramCourseEnrollmentStatuses.ACTIVE
 
+    @property
+    def course_staff(self):
```

Comment:
I think `is_course_staff` might make it clearer that this value is a boolean (similar to `is_active`), but that might just be me.